packages/editor/src/lib/editor/Editor.ts

```diff
@@ -7674,6 +7674,9 @@ export class Editor extends EventEmitter<TLEventMap> {
 		url: null,
 	}
 
+	/** @internal */
+	temporaryAssetPreview: Record<TLAssetId, string> = {}
+
```

Comment:
> make this properly private instead of @internal?

ah, yes. 👍 

> make this an immutable value stored in an atom? that way if it changes we'll cause re-renders on components depending on it automatically

i don't _think_ so. i think i don't want things to react on this changing. but let's talk more, maybe i'm missing something.